eahub/profiles/forms.py

```diff
@@ -2,8 +2,14 @@
 from django import forms
 from django.db import models
 from .models import Profile, CauseArea, GivingPledge
+from ..localgroups.models import LocalGroup
 from ..base.models import User
 
+class CustomisedMultipleModelChoiceField(forms.ModelMultipleChoiceField):
+
+    def label_from_instance(self, obj):
```

Comment:
Why do we need to override this?

Reply:
The groups were displayed as Objects (<Localgroup ...>), this way I could access their names instead. Followed [this example](https://stackoverflow.com/questions/3695754/django-customizing-display-of-modelmultiplechoicefield). 
I've now changed the name to 'CustomisedModelMultipleChoiceField', noticed that I got 'Model' and 'Multiple' the wrong way around

Reply:
We don't need this, just add a `__str__` method to the model.